undertow/WFLY-12473_UUID-instance-id.adoc

```diff
@@ -0,0 +1,65 @@
+= WFLY-12473 UUID instace-id
+:author:            Flavia Rainone
+:email:             frainone@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          instance-id,uuid,uuid-instance-id
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This feature is to be able to set the value of UndertowService.instanceId to a UUID,
+if instance-id was not defined and user desires to have it so.
+EAP6 Web Server would set it as a default whenever instance-id configuration was absent, and Undertow defaults to ${jboss.node.name} instead.
+Some customers would like to have it default to a UUID instead.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-12473[WFLY-12473]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1187
+* https://issues.jboss.org/browse/PRODMGT-1934
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[X] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+WildFly, Undertow component
+
+=== Other Interested Projects
+
+== Requirements
+
+This will be implemented as an additional attribute under the `subsystem=undertow` management model address.
+
+This will be a simple boolean attribute called `uuid-instance-id` that marks instance-id to be set equal to a uuid.
+
+=== Hard Requirements
+
+=== Nice-to-Have Requirements
+
+=== Non-Requirements
+
+== Test Plan
+
+A new test will be added to verify that the instance-id of Undertow subsystem is set to a UUID instead of ${jboss.node.name}.
+
```

Comment:
@fl4via I am posting this comment here, because I can't find the TP doc, even though it actually belongs to the "manual experiments" section of the TP doc (or to the "Integration testing", in case we have such test already);

In the linked case https://access.redhat.com/support/cases/#/case/02167248 it says that the customer's ultimate goal is `... to configure JBPSS EAP 7 to match the behavior of JBOSS EAP 6 so that the Nodes registered with Apache using an generated UUID rather than the jboss.node.name. If so, how is this done?` (btw, are they referring to mod_cluster/mod_jk used as load balancer?)

Hence I propose to have such a check there in order to be 100% to fulfill the customer's expectations;